token-metadata-creator/README.md

```diff
@@ -62,6 +64,7 @@ token-metadata-creator entry 19309eb9c066253cede617dc635223ace320ae0bbdd5bd19684
 
 | Field    | Details                                                               | Command          |
 | ---      | ---                                                                   | ---              |
+| `policy` | the exact script which hashes to the `policyId`                       | `--policy \| -p` |
```

Comment:
Yep, this is correct and should **not** be changed to match (https://github.com/cardano-foundation/cardano-token-registry/pull/296#discussion_r687291744).

The meaning of the registry entry is "The base16-encoded CBOR representation of the monetary policy script", but the description of the thing you should pass to `token-metadata-creator` is "the exact script which hashes to the policyId".